doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1444 @@
+  * Fixed a bug where various subsystems would be erroneously initialized during a failed initialization of the library.
+
+* :ref:`lib_location` library:
+
+  * Removed support for HERE location services.
+
+* :ref:`lib_at_host` library:
+
+  * Fixed a bug where AT responses would erroneously be written to the logging UART instead of being written to the chosen ``ncs,at-host-uart`` UART device when the :kconfig:option:`CONFIG_LOG_BACKEND_UART` Kconfig option was set.
+
+* :ref:`modem_info_readme` library:
+
+  * Added:
+
+    * The :c:enum:`modem_info_data_type` type for representing LTE link information data types.
+    * The :c:func:`modem_info_data_type_get` function for requesting the data type of the current modem information type.
+
+  * Deprecated the :c:func:`modem_info_type_get` function in favor of the :c:func:`modem_info_data_type_get` function.
+
+* :ref:`lib_modem_slm` library:
+
+  * Updated:
+
+    * By renaming the ``CONFIG_MODEM_SLM_WAKEUP_PIN`` and ``CONFIG_MODEM_SLM_WAKEUP_TIME`` Kconfig options to :kconfig:option:`CONFIG_MODEM_SLM_POWER_PIN` and :kconfig:option:`CONFIG_MODEM_SLM_POWER_PIN_TIME`, respectively.
+    * By renaming the ``modem_slm_wake_up`` function to :c:func:`modem_slm_power_pin_toggle`.
+
+Multiprotocol Service Layer libraries
+-------------------------------------
+
+* Added:
+
+  * Integration with the nrf2 clock control driver for the nRF54H20 SoC.
+  * Integration with Zephyr's system power management for the nRF54H20 SoC.
+  * Global domain HSFLL120 320MHz frequency request if MPSL is enabled.
+    The high frequency in global domain is required to ensure that fetching instructions from L2-cache and MRAM is as fast as possible.
+    It is needed for the radio protocols to operate correctly.
+  * MRAM always-on request for scheduled radio events.
+    It is needed to avoid MRAM wake-up latency for radio protocols.
+
+Libraries for networking
+------------------------
+
+* Added:
+
+  * The :ref:`lib_downloader` library.
+  * A backend for the :ref:`TLS Credentials Subsystem <zephyr:sockets_tls_credentials_subsys>` that stores the credentials in the modem, see :kconfig:option:`CONFIG_TLS_CREDENTIALS_BACKEND_NRF_MODEM`.
+
+* Deprecated the :ref:`lib_download_client` library.
+  See the :ref:`migration guide <migration_3.0_recommended>` for recommended changes.
+
+* Updated the following libraries to use the :ref:`lib_downloader` library instead of the :ref:`lib_download_client` library:
+
+  * :ref:`lib_nrf_cloud`
+  * :ref:`lib_aws_fota`
+  * :ref:`lib_azure_fota`
+  * :ref:`lib_fota_download`
+
+* :ref:`lib_nrf_cloud_pgps` library:
+
+  * Fixed the warning due to missing ``https`` download protocol.
+
+* :ref:`lib_fota_download` library:
+
+  * Added error codes related to unsupported protocol, DFU failures, and invalid configuration.
+  * Updated to use the :ref:`lib_downloader` library for CoAP downloads.
```

Comment:
this line is redundant, already mentioned in 1253

Reply:
Removed the redundant entry.